pkg/metrics/querynode_metrics.go

```diff
@@ -51,7 +51,7 @@ var (
 		prometheus.HistogramOpts{
 			Namespace: milvusNamespace,
 			Subsystem: typeutil.QueryNodeRole,
-			Name:      "process_insert_or_delete_ms",
+			Name:      "process_insert_or_delete_latency",
```

Comment:
changing name will cause all legacy dashboard data missing, any particular reason having this?